abci/types/application.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 )
 
+//go:generate ../../scripts/mockery_generate.sh Application
+//
```

Comment:
```suggestion

```

I suggest separating these so the generate comment doesn't show up in the godoc.